Domain.Testing/SqlQueryReservationService.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Its.Domain.Sql;
+using System;
+using System.Data.Entity;
+using System.Threading.Tasks;
+
+namespace Microsoft.Its.Domain.Testing
+{
+    public class SqlQueryReservationService : IQueryReservationService
+    {

```

Comment:
This name is a little confusing. Maybe `SqlReservationQuery : IReservationQuery`?